ripple/src/main/webapp/businessdetails.js

```diff
@@ -1,7 +1,63 @@
 /* When user clicks on a team member, add highlight to that avatar */
-$(document).ready(function(){
+$(document).ready(() => {
   $(".bd-avatar").click(function() {
     $(".bd-avatar").removeClass("bd-team-highlight");
-    $(this).addClass('bd-team-highlight');
-   });
-  });
\ No newline at end of file
+    $(this).addClass("bd-team-highlight");
+  });
+});
+
+$(".selector").on("mouseover", function () {
+    //stuff to do on mouseover
```

Comment:
Can this be an inline function?

Reply:
Deleted function itself b/c it is not being used.